onadata/apps/api/viewsets/user_profile_viewset.py

```diff
@@ -154,10 +155,14 @@ def change_password(self, request, user):  # pylint: disable=W0613
         current_password = request.data.get('current_password', None)
         new_password = request.data.get('new_password', None)
 
+        metadata = user_profile.metadata or {}
```

Comment:
Why not have the the newly introduced `metadata` variable within the `if new_password` block? Is it used anywhere else outside the block?